src/Surfnet/StepupMiddleware/ApiBundle/Doctrine/Type/UseRaLocationsOptionType.php

```diff
@@ -61,14 +61,20 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
             return $value;
         }
 
+        // Prevent a too lenient boolean conversion on non-postgresql platforms
+        if ($platform->getName() !== 'postgresql') {
+            if ($value !== 1 && $value !== 0 && $value !== '1' && $value !== '0' && !is_bool($value)) {
```

Comment:
could that string be converted? I mean, how would it get into the database? Oh, and what about [oracle][1]? KISS - we do not support anything but MySQL (MariaDB).

[1]: https://docs.oracle.com/cd/B19306_01/olap.102/b14346/dml_datatypes004.htm#CJACGBGE